[2024-09-25T11:12:11.048+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-25T11:12:11.086+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_pipeline.pipeline.warehouse.step_1.load.categories manual__2024-09-25T11:08:58.108660+00:00 [queued]>
[2024-09-25T11:12:11.113+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_pipeline.pipeline.warehouse.step_1.load.categories manual__2024-09-25T11:08:58.108660+00:00 [queued]>
[2024-09-25T11:12:11.113+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2024-09-25T11:12:11.146+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): pipeline.warehouse.step_1.load.categories> on 2024-09-25 11:08:58.108660+00:00
[2024-09-25T11:12:11.164+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=980) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-09-25T11:12:11.168+0000] {standard_task_runner.py:72} INFO - Started process 985 to run task
[2024-09-25T11:12:11.170+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etl_pipeline', 'pipeline.warehouse.step_1.load.categories', 'manual__2024-09-25T11:08:58.108660+00:00', '--job-id', '243', '--raw', '--subdir', 'DAGS_FOLDER/etl_pipeline/run.py', '--cfg-path', '/tmp/tmpimmmh_oq']
[2024-09-25T11:12:11.174+0000] {standard_task_runner.py:105} INFO - Job 243: Subtask pipeline.warehouse.step_1.load.categories
[2024-09-25T11:12:11.330+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_pipeline.pipeline.warehouse.step_1.load.categories manual__2024-09-25T11:08:58.108660+00:00 [running]> on host e10075799413
[2024-09-25T11:12:11.584+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_pipeline' AIRFLOW_CTX_TASK_ID='pipeline.warehouse.step_1.load.categories' AIRFLOW_CTX_EXECUTION_DATE='2024-09-25T11:08:58.108660+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-09-25T11:08:58.108660+00:00'
[2024-09-25T11:12:11.588+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-25T11:12:11.619+0000] {base.py:84} INFO - Retrieving connection 'minio'
[2024-09-25T11:12:11.668+0000] {base.py:84} INFO - Retrieving connection 'warehouse_db'
[2024-09-25T11:12:11.721+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-09-25T11:12:11.753+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-25T11:12:11.766+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=etl_pipeline, task_id=pipeline.warehouse.step_1.load.categories, run_id=manual__2024-09-25T11:08:58.108660+00:00, execution_date=20240925T110858, start_date=20240925T111211, end_date=20240925T111211
[2024-09-25T11:12:11.830+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-09-25T11:12:11.920+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-09-25T11:12:11.923+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
